girder/molecules/molecules/utilities/molecules.py

```diff
@@ -80,11 +80,18 @@ def create_molecule(data_str, input_format, user, public, gen3d=True,
             'provenance': provenance
         }
 
-        # Set a name if we find one
-        name = chemspider.find_common_name(inchikey)
+        # Set a name if we find one or one is provided
+        if 'name' in parameters:
+            name = parameters['name']
+        else:
+            name = chemspider.find_common_name(inchikey)
         if name is not None:
             mol_dict['name'] = name
 
+        # Set a wikipedia link if one is provided
+        if 'wikipedia' in parameters:
+            mol_dict['wiki'] = parameters['wikipedia']
```

Comment:
Lets call this `wikipediaUrl'

Reply:
Rename just the input parameters key(`wikipedia`), the molecule key(`wiki`), or both?

Reply:
I would say both.

Reply:
Fixed for both. Also updated in openchemistry/mongochemclient#166